Source/LibationAvalonia/Program.cs

```diff
@@ -55,9 +53,25 @@ static void Main(string[] args)
 			if (Configuration.IsWindows)
 				AppScaffolding.LibationScaffolding.SetReleaseIdentifier(AppScaffolding.ReleaseIdentifier.WindowsAvalonia);
 			else if (Configuration.IsLinux)
-				AppScaffolding.LibationScaffolding.SetReleaseIdentifier(AppScaffolding.ReleaseIdentifier.LinuxAvalonia);
+			{
```

Comment:
This could be cleaned up by refactoring to the OS specific libraries. If you don't think it would divorce it too much from the logic flow here. To me, this is a lot cleaner and equally expressive. Just a matter of taste though; not a mandate.

    AppScaffolding.LibationScaffolding.SetReleaseIdentifier(osconfig.GetReleaseIdentifier());